nncf/torch/quantization/metrics.py

```diff
@@ -291,11 +291,11 @@ def collect(self) -> QuantizationConfigurationStatistics:
                         self._marking_edges(merged_original_graph, node_key, queue)
                     else:
                         is_op_non_change_precision_activation_tensor = True
-                        node_op_name = node[NNCFGraph.NODE_TYPE_ATTR]
-                        for op in DEFAULT_PT_QUANT_TRAIT_TO_OP_DICT[QuantizationTrait.INPUTS_QUANTIZABLE]:
-                            if node_op_name in op.get_all_aliases():
```

Comment:
I wonder why it was so complicated logic before and so easy now. Don't we really miss anything here? :)

Reply:
Looks like somebody used `type` of nncfNode to match metatypes instead of using available `metatype` of nncfNode.
`get_all_aliases()` just translates `metatype` to `type` so I just skip that translation and search

Reply:
We can ask the [author](@kshpv) :)